src/VAC/VectorAnimationComplex/InfillPattern.cpp

```diff
@@ -494,7 +494,7 @@ void InfillPattern::setPattern(InfillPattern::Pattern pattern)
     pattern_ = pattern;
 }
 
-bool InfillPattern::applyInfill() const
+bool InfillPattern::isApplyInfill() const
```

Comment:
I still find the `is` prefix odd. Is that standard Qt? I can't find it in the style guide but seems to be common naming in Qt. `shouldApplyInfill` or `isApplyingInfill` make more sense grammatically at least.  But I guess it's fine either way.

Reply:
Yeah, it's just grammar. isInfillEnabled() might be better?